plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/GitHookTasksTest.kt

```diff
@@ -138,6 +138,29 @@ class GitHookTasksTest : AbstractPluginTest() {
         }
     }
 
+    @Test
+    internal fun `Format hook should format files when they are renamed`() {
+        projectRoot.setupGradleProject()
+        val gitDir = projectRoot.initGit()
+
+        build(":$INSTALL_GIT_HOOK_FORMAT_TASK").run {
+            assertThat(task(":$INSTALL_GIT_HOOK_FORMAT_TASK")?.outcome).isEqualTo(TaskOutcome.SUCCESS)
+            // assertThat(gitDir.preCommitGitHook().readText()).matches(""".*awk '.* \$NF ~ \/.*\/ \{ print \$NF \}'.*""")
```

Comment:
Is this commented line needed?

Reply:
I was trying to match that regex instead of just that text. Could you fix it, or would you rather I just remove it.

Reply:
I've removed the commented code